### PR TITLE
feat: added missing api's for weightage

### PR DESCRIPTION
**Pull request by Abbaskt**

### CODE REVIEW

**Review by chetankashetti — APPROVED** [review]

File: docs/api-weightages/index.md [hunk 1]
```diff
@@ -9,11 +9,15 @@ The number of requests you could make every month is appropriately limited in al
 Path &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; | Weightage
 --- | ---
 /v1/:chain/address/:address/assets | 20
+/v1/batch-assets | 1000
 /v2/:chain/address/:address/userData | 50
 /v1/:chain/address/:address/transactions | 5
-/v3/:chain/address/:address/transactions | 5
 /v2/:chain/address/:address/transactions | 20
+/v3/:chain/address/:address/transactions | 5
+/v1/:chain/address/:address/transactions/count | 50
```
Comment (suyog-bhat) [inline docs/api-weightages/index.md:17]: "/v1/:chain/address/:address/transactions/count": {
    "Weightage": 10,
    "tag": "core"
  },

File: docs/api-weightages/index.md [hunk 2]
```diff
@@ -9,11 +9,15 @@ The number of requests you could make every month is appropriately limited in al
...
+/v1/:chain/token/:tokenAddress/transactions | 10
```
[the remark this answers is not in the document]
Comment (suyog-bhat) [reply docs/api-weightages/index.md:18]: "/v1/:chain/token/:address/transactions": {
    "Weightage": 30,
    "tag": "core"
  },

**Review by suyog-bhat — CHANGES_REQUESTED** [review]

File: docs/api-weightages/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Abbaskt) [reply docs/api-weightages/index.md:18]: These have been updated in api service

File: docs/api-weightages/index.md (hunk 1 above)
Comment (Abbaskt) [reply docs/api-weightages/index.md:17]: These have been updated in api service

**Pull request merged**